library/init/meta/interactive.lean

```diff
@@ -6,8 +6,11 @@ Authors: Leonardo de Moura
 prelude
 import init.meta.tactic init.meta.rewrite_tactic init.meta.simp_tactic
 import init.meta.smt.congruence_closure init.category.combinators
+import init.meta.lean_parser
 
 namespace interactive
+@[reducible] meta def parse {α : Type} (p : lean_parser α) : Type := α
```

Comment:
> Re: has_to_pexpr. It does something different on pre-expressions than has_quote. The has_quote instance would be mk_quote.

It is the identity function.
```lean
meta instance : has_to_pexpr pexpr :=
⟨id⟩
```
Note that
```lean
λ e : pexpr, `(%%e)
```
is 
```lean
λ e, pexpr.subst (fun _x, x) (pexpr e)
```
which is essentially the identity function too.